pkg/cmd/pipelineresource/delete.go

```diff
@@ -82,7 +83,33 @@ func deleteResources(s *cli.Stream, p cli.Params, preNames []string) error {
 	d := deleter.New("PipelineResource", func(resourceName string) error {
 		return cs.Resource.TektonV1alpha1().PipelineResources(p.Namespace()).Delete(resourceName, &metav1.DeleteOptions{})
 	})
-	d.Delete(s, preNames)
-	d.PrintSuccesses(s)
+	if deleteAll {
+		preNames, err = allPipelineResourceNames(p, cs)
+		if err != nil {
+			return err
+		}
+		d.Delete(s, preNames)
+	} else {
+		d.Delete(s, preNames)
```

Comment:
i think you may want to make 91 and 93 Deletes as only one?

Reply:
Yeah, good call!